handy-pandas/djangazon
# Am my product view

## Description
1. Name of Branch:
```am-my-product-view```

2. Tickets related to PR (with links):
[User can Remove Sale post](https://github.com/handy-pandas/djangazon/issues/19)
[User can view status of products uploaded](https://github.com/handy-pandas/djangazon/issues/15)

3. Feature/Fix Description:
Incorporated quantity on product order table by adjusting how productorder table is used in buying and displaying products you are selling.

4. What architectual changes made:
added view for my products page

5. Any dependency changes:
No, but please use
 ```python manage.py deletedb```
 ```python manage.py builddb```

## Testing
### To Test:
- Go through the process of creating a products with a high quantity (100 will work)
- Make sure quantity shows on my products page
- Buy product and make sure quantity on my products page goes down by the quantity you bought (currently it is the number of times you click on 'add to order' that is the quantity of how many you are buying)
- Quantity will show on shopping cart page as you how many are on your order

1. New Unit Tests:
no

2. All Tests Pass:
yes

## Documentation
1. Fully completed Documentation with signature:
yes

2. Readme file updated:
no

## Deployment
1. Any new migrations to run:
```python manage.py deletedb```
 ```python manage.py builddb```
